packages/evm/src/types.ts

```diff
@@ -346,6 +346,11 @@ export interface EVMOpts {
    *
    */
   profiler?: EVMProfilerOpts
+
+  /**
+   * Must be present if consensus type is clique/poa, else error will be thrown
+   */
+  cliqueSigner?: (header: Block['header']) => Address
```

Comment:
Ah, nice trick!

Reply:
typing wizard @acolytec3 ftw

Reply:
Can I add this to my official job title?